### ISSUE TITLE
Upload throws notification instead of uploading.

### ISSUE THREAD

**Issue by BetaBugish** [c9/c9.ide.upload#1]
I've stumbled across a bug that keeps me from uploading some spritesheets to a cloud9 workspace over at c9.io.

This occurs when simply dragging a file to the file tree on the left in the IDE.
![screenshot 2016-01-14 at 20 08 23](https://cloud.githubusercontent.com/assets/6827387/12335943/0092d7be-bb02-11e5-9cad-fee5c52844f7.png)
![screenshot 2016-01-14 at 20 08 43](https://cloud.githubusercontent.com/assets/6827387/12335944/00a006aa-bb02-11e5-9d42-1d4c6e0111e6.png)

You might notice I'm using ChromeOS dev channel version 49.0.2611.0, ( I can try a more stable version of ChromeOS monday at work).

I've set up my own version of c9/core and the same issue also happens.
![screenshot 2016-01-14 at 20 45 27](https://cloud.githubusercontent.com/assets/6827387/12336023/63fc3818-bb02-11e5-8548-19bbff5041e6.png)

**Comment by BetaBugish** [comment]
Upon further examination of the plugin, I've been able to squeeze out this message:
![screenshot 2016-01-14 at 21 22 00](https://cloud.githubusercontent.com/assets/6827387/12336445/f8ec1c20-bb04-11e5-9f60-b92924e95ebf.png)

It is being returned by the entry.file call here:
https://github.com/c9/c9.ide.upload/blob/master/batch.js#L142

**Comment by nightwing** [comment]
This looks like a ChromeOS specific bug, i do not see it on chromium 49.0.2622.0 on windows.

**Comment by BetaBugish** [comment]
Yes, you confirm what I thought. I've just found that it just fails when uploading something from the "Google Drive mount" in the ChromeOS file manager, local files work just fine.
I'll check if it happens on more stable versions of ChromeOS too. If it doesn't this should probably be reported to the people over at ChromiumOS.
